hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CombinedBlockCache.java

```diff
@@ -79,7 +79,13 @@ public Cacheable getBlock(BlockCacheKey cacheKey, boolean caching,
       boolean repeat, boolean updateCacheMetrics) {
     // TODO: is there a hole here, or just awkwardness since in the lruCache getBlock
     // we end up calling l2Cache.getBlock.
-    return lruCache.containsBlock(cacheKey)?
+    boolean existInL1 = lruCache.containsBlock(cacheKey);
+    if (!existInL1 && updateCacheMetrics) {
+      // If the block does not exist in L1, this check should be counted as a miss.
+      combinedCacheStats.lruCacheStats
+        .miss(caching, cacheKey.isPrimary(), cacheKey.getBlockType());
+    }
+    return existInL1?
```

Comment:
nit, space between `existInL1` and `?`

Reply:
The former line is without space, should I add one here?

Reply:
rectified in the last commit.